crates/sparrow-compiler/src/functions/pushdown.rs

```diff
@@ -110,7 +110,8 @@ impl Pushdown {
             | DataType::Duration(_)
             | DataType::Interval(_)
             | DataType::Utf8
-            | DataType::LargeUtf8 => {
+            | DataType::LargeUtf8
+            | DataType::Map(..) => {
```

Comment:
Probably need to rename this, since it's currently `PrimitivePushdown`, but :shrug: